src/deluge/gui/menu_item/patched_param.h

```diff
@@ -31,6 +31,7 @@ class PatchedParam : public Param, public MenuItemWithCCLearning {
 	PatchedParam() = default;
 	PatchedParam(int32_t newP) : Param(newP) {}
 	MenuItem* selectButtonPress();
+	ActionResult buttonAction(deluge::hid::Button b, bool on) { return Param::buttonAction(b, on); }
```

Comment:
I think this should just pass through inheritance?

Reply:
> I think this should just pass through inheritance?

I tried removing the three button action calls and it did not work anymore.

I treated it the same as the selectButtonAction function which is also added to all three of these classes which inherit param.